tests/templates/test_state_preparations/test_state_prep_mps.py

```diff
@@ -27,6 +27,7 @@
 
 class TestMPSPrep:
 
+    @pytest.mark.jax
```

Comment:
Should we somehow mark all `assert_valid` tests with `pytest.mark.jax`? That seems like something annoying to do, but more importantly we need a way to enforce it going forward.